### DISCUSSION TITLE
Add the functionality to send images with the request

### DISCUSSION THREAD

**Discussion by Wild3d** [acheong08/EdgeGPT#585] [Ideas]
Appending images to your prompt would be a great feature to have, since it seems that they have rolled out the feature of BingChat.

![Screenshot 2023-06-30 163800](https://github.com/acheong08/EdgeGPT/assets/38893286/a64cf025-1897-4b66-9ba4-a73342823236)

**Comment by ustedestaaqui** [comment]
Yes

**Comment by acheong08** [comment]
I don't have access to this feature it seems

**Reply by Wild3d** [reply]
I used to have access to the feature till Monday (11/07) but it seem that they have pulled it again from the service.

**Comment by bi1101** [comment]
Seems like this feature is rolled out for everyone now
